packages/@lwc/integration-karma/scripts/karma-plugins/lwc.js

```diff
@@ -107,7 +108,6 @@ function createPreprocessor(config, emitter, logger) {
             // also adding the source map inline for browser debugging.
             // eslint-disable-next-line require-atomic-updates
             file.sourceMap = map;
-            code + `\n//# sourceMappingURL=${map.toUrl()}\n`;
```

Comment:
This was a mistake I imagine. (Note `code` is not actually reassigned.) In any case, the `sourceMappingURL` is already in the `code`, so this is unnecessary.